internal/app/api/v1/executions.go

```diff
@@ -176,9 +177,27 @@ func (s TestkubeAPI) notifyEvents(eventType *testkube.WebhookEventType, executio
 		})
 	}
 
+	s.notifySlack(eventType, execution)
```

Comment:
for first iteration it's ok, 

but I was thinking more about doing whole EventsEmitter pluggable 
e.g. 
```
		s.EventsEmitter.Notify(testkube.WebhookEvent{
			Uri:       wh.Spec.Uri,
                        Kind:      "SLACK"  // it would be defined as Webhook kind == SLACK 
			Type_:     eventType,
			Execution: &execution,
		})
```

Reply:
Don't know yet what full implemetation could be provided but we could add several Kinds - like slack or other internal event types if needed.